class Exc1 
{
static void subroutine() 
{
int d = 0;
int a = 10 / d;
}
public static void main(String args[]) 
{
Exc1.subroutine();
}
}


The resulting stack trace from the default exception handler shows how the entire call
stack is displayed:
java.lang.ArithmeticException: / by zero
at Exc1.subroutine(Exc1.java:4)
at Exc1.main(Exc1.java:7)